app/src/main/java/com/kickstarter/ui/fragments/BackingFragment.kt

```diff
@@ -235,6 +241,20 @@ class BackingFragment : Fragment() {
 
             val boldPortionLength = text.toString().split(".").first().length
             setBoldSpanOnTextView(boldPortionLength, this, resources.getColor(R.color.kds_support_400, null))
+
+            binding?.paymentScheduleComposeView?.setContent {
```

Comment:
My inkling here is that this will appear on all manage pledge screens, regardless of if they are a plot pledge. We should probably gate this with the condition that it only shows if pledge is PLOT.

Reply:
Agre with @leighdouglas